alpine.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:3.1.4-alpine3.19 AS base
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    TZ=Europe/London \
```

Comment:
I'm wondering if it's possible to inherit the timezone from the host? That seems like the most intuitive for the developer.
(Also note that London is not always the same as GMT/UTC due to daylight savings)

Reply:
I was just following the timezone that was set in the original `Dockerfile`. It looks like there is a way to sync to the host by mounting the local timezone file in `compose.yaml`, which I implemented.

Reply:
Nice one! It looks as though it's mapping to a *nix system file, so I guess won't work on Windows? (even so, it's still a helpful enhancement for non-windows users).

Reply:
Ahh, true. As always, Windows has to be the black sheep. I googled around for a possible solution, and the short story is that Windows's timezones are completely messed up. Anyway, we got close.

Reply:
😆 True, thanks for following up